wp-tevko-responsive-images.php

```diff
@@ -209,9 +209,19 @@ function tevkori_get_srcset_array( $id, $size = 'thumbnail' ) {
 	// Calculate the image aspect ratio.
 	$img_ratio = $img_height / $img_width;
 
+	// If the image has been cropped, it will contain a unique hash.
+	// We look for that hash and use it later to filter out images that aren't
+	//  of the same crop.
+	$img_cropped = preg_match( '/e\d{13}/', $img_url, $img_crop_hash );
```

Comment:
WP core uses ```'/-e[0-9]{13}\./'``` (wp-admin/includes/image-edit.php) so maybe it's good to use the same regex.